test/fixtures/o-test/.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "ecmaFeatures": {
+    "modules": true
+  },
+  "env": {
+    "es6": true,
+    "browser": true
+  },
+  "rules": {
+    "eqeqeq": 2,
+    "guard-for-in": 2,
+    "no-extend-native": 2,
+    "wrap-iife": 2,
+    "new-cap": 2,
+    "no-caller": 2,
+    "no-irregular-whitespace": 1,
+    "no-trailing-spaces": 2,
+    "no-multi-str": 0,
+    "dot-notation": 0,
+    "strict": [2, "function"],
+    "quotes": [1, "single"],
```

Comment:
Very much in favour of removing the warning, it's quite noisy. I wanted to test it out, it's quite nice to have some kind of convention on modules, but after running verify on a few modules, it spilled out dozens and dozens of warnings. I'll probably keep it in obt though